README.md

```diff
@@ -27,10 +27,16 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 * We recommend you install the [free Anaconda Python distribution](https://www.continuum.io/downloads), which will automatically install Python along with all of the machine learning and scientific computing libraries you need.
 
+* It's a good idea to learn Python's syntax, here are some resources we've used and recommend for learning 'Python for Data Science':
+  
+  - [CodeAcademy](https://www.codecademy.com/learn/python)
+  - [PyData Tutorial](https://www.youtube.com/watch?v=SWjnth2_JRY) and associated [notebook](https://github.com/gumption/Python_for_Data_Science/blob/master/3_Python_Basic_Concepts.ipynb)
```

Comment:
maybe change link text to "Python for Data Science Tutorial @ PyData Seattle 2015"